assets/react/front/course/_lesson.js

```diff
@@ -9,10 +9,18 @@ window.jQuery(document).ready($=>{
 
     $(document).on('click', '.tutor-single-course-lesson-comments button[type="submit"]', function(e){
         e.preventDefault();
+
         const {__} = wp.i18n;
         let btn = $(this);
         let form = btn.closest('form');
         let data = form.serialize();
+        let comment = form.find('textarea[name="comment"]').val();
+        
+        if (comment.length === 0) {
```

Comment:
I think we should `trim` before checking to prevent empty spaces or tabs like below:
`comment.trim().length === 0`